custom_addons/hosting/static/src/scss/expiry_countdown_systray.scss

```
@@ -1,5 +1,10 @@
 .o_hosting_expiry_countdown_systray {
     font-weight: 600;
+    // Replicates Bootstrap's `.badge` font-size/line-height/white-space (_badge.scss), now
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)